pk32 build error

Run the following command:
```
./configure --prefix=/workspace/tonglin/release/ --enable-multilib  --with-sim=spike
make newlib buid-sim
```
Then, got following error:
```
/workspace/tonglin/release/riscv-toolchain/bin/../lib/gcc/riscv64-unknown-elf/11.1.0/../../../../riscv64-unknown-elf/bin/ld: failed to merge target specific data of file /workspace/tonglin/release/riscv-toolchain/bin/../lib/gcc/riscv64-unknown-elf/11.1.0/libgcc.a(div.o)
riscv64-unknown-elf-objcopy -S -O binary --change-addresses -0x80000000 bbl bbl.bin
collect2: error: ld returned 1 exit status
make[1]: *** [Makefile:336: pk] Error 1
make[1]: Leaving directory '/workspace/tonglin/projects/riscv-gnu-toolchain/build/build-pk32'
make: *** [Makefile:825: stamps/build-pk32] Error 2
```

Add **"--with-abi=ilp32d"** option to target "stamps/build-pk32" can fix it
```
stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
	rm -rf $@ $(notdir $@)
	mkdir $(notdir $@)
	cd $(notdir $@) && $</configure \
		--prefix=$(INSTALL_DIR) \
		--host=$(NEWLIB_TUPLE) \
                --with-abi=ilp32d \
		--with-arch=rv32gc
	$(MAKE) -C $(notdir $@)
	cp $(notdir $@)/pk $(INSTALL_DIR)/$(NEWLIB_TUPLE)/bin/pk32
	mkdir -p $(dir $@)
	date > $@
```

> Run the following command:
> 
> ```
> ./configure --prefix=/workspace/tonglin/release/ --enable-multilib  --with-sim=spike
> make newlib buid-sim
> ```
> 
> Then, got following error:
> 
> ```
> /workspace/tonglin/release/riscv-toolchain/bin/../lib/gcc/riscv64-unknown-elf/11.1.0/../../../../riscv64-unknown-elf/bin/ld: failed to merge target specific data of file /workspace/tonglin/release/riscv-toolchain/bin/../lib/gcc/riscv64-unknown-elf/11.1.0/libgcc.a(div.o)
> riscv64-unknown-elf-objcopy -S -O binary --change-addresses -0x80000000 bbl bbl.bin
> collect2: error: ld returned 1 exit status
> make[1]: *** [Makefile:336: pk] Error 1
> make[1]: Leaving directory '/workspace/tonglin/projects/riscv-gnu-toolchain/build/build-pk32'
> make: *** [Makefile:825: stamps/build-pk32] Error 2
> ```
> 
> Add **"--with-abi=ilp32d"** option to target "stamps/build-pk32" can fix it
> 
> ```
> stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
> 	rm -rf $@ $(notdir $@)
> 	mkdir $(notdir $@)
> 	cd $(notdir $@) && $</configure \
> 		--prefix=$(INSTALL_DIR) \
> 		--host=$(NEWLIB_TUPLE) \
>                 --with-abi=ilp32d \
> 		--with-arch=rv32gc
> 	$(MAKE) -C $(notdir $@)
> 	cp $(notdir $@)/pk $(INSTALL_DIR)/$(NEWLIB_TUPLE)/bin/pk32
> 	mkdir -p $(dir $@)
> 	date > $@
> ```

I tried to reproduce this but get a different failure and, for some reason, the `pk` directory is empty (including no hidden items such as `.git`):
```
git clone https://github.com/riscv/riscv-gnu-toolchain multilib-spike
cd multilib-spike
./configure --prefix=`pwd`/installed-tools --enable-multilib --with-sim=spike

...

make[1]: Leaving directory '/home/tommy/Downloads/riscv-tools/multilib-spike/build-spike'
mkdir -p stamps/
date > stamps/build-spike
rm -rf stamps/build-pk32 build-pk32
mkdir build-pk32
cd build-pk32 && /home/tommy/Downloads/riscv-tools/multilib-spike/pk/configure \
	--prefix=/home/tommy/Downloads/riscv-tools/multilib-spike/installed-tools \
	--host=riscv64-unknown-elf \
	--with-arch=rv32gc
/bin/sh: 1: /home/tommy/Downloads/riscv-tools/multilib-spike/pk/configure: not found
make: *** [Makefile:825: stamps/build-pk32] Error 127

ls -al pk
total 8
drwxrwxr-x  2 tommy tommy 4096 Oct  5 13:47 .
drwxrwxr-x 30 tommy tommy 4096 Oct  5 17:22 ..
```
Branch: main
Commit: c2857f9c03867f9fe9ee8e44b7cf57dd83020455
Build log: [build.log.tar.gz](https://github.com/riscv-collab/riscv-gnu-toolchain/files/9717376/build.log.tar.gz)

--- a/Makefile.in
+++ b/Makefile.in
@@ -825,7 +825,8 @@ stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
 	cd $(notdir $@) && $</configure \
 		--prefix=$(INSTALL_DIR) \
 		--host=$(NEWLIB_TUPLE) \
-		--with-arch=rv32i
+		--with-arch=rv32imafdc \
+		--with-abi=ilp32f
 	$(MAKE) -C $(notdir $@)
 	cp $(notdir $@)/pk $(INSTALL_DIR)/$(NEWLIB_TUPLE)/bin/pk32
 	mkdir -p $(dir $@)

--- a/Makefile.in
+++ b/Makefile.in
@@ -825,7 +825,7 @@ stamps/build-pk32: $(PK_SRCDIR) stamps/build-gcc-newlib-stage2
 	cd $(notdir $@) && $</configure \
 		--prefix=$(INSTALL_DIR) \
 		--host=$(NEWLIB_TUPLE) \
-		--with-arch=rv32gc
+		--with-arch=rv32i
 	$(MAKE) -C $(notdir $@)
 	cp $(notdir $@)/pk $(INSTALL_DIR)/$(NEWLIB_TUPLE)/bin/pk32
 	mkdir -p $(dir $@)

